misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/TesterinaRegistry.java

```diff
@@ -17,7 +17,7 @@
 */
 package org.ballerinalang.testerina.core;
 
-import org.ballerinalang.test.runtime.entity.TestSuite;
+import io.ballerina.projects.testsuite.TestSuite;
```

Comment:
shall we remove the `io.ballerina.projects.testsuite` package and move all the classes back to testerina runtime module?